include/olt-manager.php

```diff
@@ -56,184 +29,52 @@ public function __construct() {
 			return;
 		}
 
-		// Saves the default locale before we start any language manipulation
-		$this->default_locale = get_locale();
+		// Filters for text domain management.
+		add_filter( 'load_textdomain_mofile', array( $this, 'bypass_load_textdomain_mofile' ) );
 
-		// Filters for text domain management
-		add_filter( 'load_textdomain_mofile', array( $this, 'load_textdomain_mofile' ), 10, 2 );
-		add_filter( 'gettext', array( $this, 'gettext' ), 10, 3 );
-		add_filter( 'gettext_with_context', array( $this, 'gettext_with_context' ), 10, 4 );
-
-		// Loads text domains
-		add_action( 'pll_language_defined', array( $this, 'load_textdomains' ), 2 ); // After PLL_Frontend::pll_language_defined
+		// Loads text domains.
+		add_action( 'pll_language_defined', array( $this, 'load_textdomains' ), 2 ); // After PLL_Frontend::pll_language_defined.
 		add_action( 'pll_no_language_defined', array( $this, 'load_textdomains' ) );
 	}
 
 	/**
-	 * Access to the single instance of the class
-	 *
-	 * @since 1.7
-	 *
-	 * @return PLL_OLT_Manager
-	 */
-	public static function instance() {
-		if ( empty( self::$instance ) ) {
-			self::$instance = new self();
-		}
-
-		return self::$instance;
-	}
```

Comment:
I propose to keep this method and keep using it in the production code (to keep access to the filters if some dev want to remove them). I did not remember but this is not a true singleton, so we can keep creating new instances in tests.

Reply:
Done in 3a60b577.